### PR TITLE
So/fix padding all pages

### LINKED ISSUE #111: Header buttons are incorrectly positioned
Eg: "Oxymore" and "Menu"

image

Check across the different pages, it seems to differ

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?

- no changes, just added the `padding: 24px` back in to Global Style because I accidentally removed it through resolving merge conflicts. face palm

### Which issue(s) does this PR fix?

Fixes #111 

### How to test
same as before, just check all the pages display with the same `24px` padding

### CODE REVIEW

File: src/GlobalStyle.tsx [hunk 1]
```diff
@@ -8,18 +8,20 @@ const GlobalStyle = createGlobalStyle`
   body {
     min-height: 100%;
     height: 100%;
-    font-family: "Helvetica Neue";
+    font-family: "Helvetica Neue", Sans-Serif;
     text-decoration: none;
     color: ${theme.colors.athensGray};
     background: ${theme.colors.black};
+    padding: 24px;
```
Comment (mantagen) [inline src/GlobalStyle.tsx:15]: I'd probably suggest using `${theme.space[6]}` here

**Review by mantagen — APPROVED** [review]
Yayy, just one little recommendation!

File: src/GlobalStyle.tsx (hunk 1 above)
Comment (sssoja) [reply src/GlobalStyle.tsx:15]: yeee this is nice but weirdly it doesn't work for me :(

Screenshot 2020-09-22 at 18 59 30

Comment (mantagen) [reply src/GlobalStyle.tsx:15]: Ah yeah, `padding: ${theme.space[6]}px;`

**Pull request merged**